library/manageiq_policy.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/python
+
+import os
+from ansible.module_utils.basic import *
```

Comment:
I've seen both forms of imports (on top and in the bottom), I don't really have a strong preference here. @cben what do you think?

```
if __name__ == "__main__":
    main()
```
allows the imports you're reffering to and I'm already testing the modules in `tests/` (should improve of course)